css/style.css

```diff
@@ -748,11 +760,14 @@ ul.main-menu:not(.off-canvas) li a.active:before {
   width: 50%;
   align-self: center;
 }
+
 .section--inspiration .title h1 {
   font-family: "Lora";
   font-weight: 400;
-  font-size: var(--font-xl);
+  font-size: 78px;
+  line-height: 90px;
 }
```

Comment:
@Right-Noe can you please stick to using rems as font size unit and either integer or per cent for line heights. It will be easier to address responsive layout for smaller devices.